vm/rust/src/lib.rs

```diff
@@ -98,26 +103,16 @@ pub extern "C" fn cairoVMCall(
         class_hash: class_hash,
         code_address: None,
         caller_address: ContractAddress::default(),
-        initial_gas: INITIAL_GAS_COST,
+        initial_gas: get_versioned_constants(block_info.version).gas_cost("initial_gas_cost"),
     };
 
-    const GAS_PRICES: GasPrices = GasPrices {
-        eth_l1_gas_price: 1,
-        strk_l1_gas_price: 1,
-    };
-    let mut state = CachedState::new(reader, GlobalContractCache::default());
+    let mut state = CachedState::new(reader, GlobalContractCache::new(1));
```

Comment:
Q: what is the GlobalContractCache? Also, do we pass in 1, I thought we used 100 before?